packages/evm/src/evm.ts

```diff
@@ -137,7 +138,23 @@ export class EVM implements EVMInterface {
 
   protected readonly _emit: (topic: string, data: any) => Promise<void>
 
-  constructor(opts: EVMOpts = {}) {
+  private bn128: {
+    ec_pairing: (input_str: string) => string
+    ec_add: (input_str: string) => string
+    ec_mul: (input_hex: string) => string
+  }
+
+  static async create(opts: EVMOpts) {
```

Comment:
I think we should make `opts` optional.
```suggestion
  static async create(opts?: EVMOpts) {
```